http_request_randomizer/requests/proxy/requestProxy.py

```diff
@@ -37,9 +37,9 @@ def __init__(self, web_proxy_list=[], sustain=False):
         # Each of the classes below implements a specific URL Parser
         #####
         parsers = list([])
-        parsers.append(FreeProxyParser('http://free-proxy-list.net'))
-        parsers.append(ProxyForEuParser('http://proxyfor.eu/geo.php', 1.0))
-        parsers.append(RebroWeeblyParser('http://rebro.weebly.com'))
+        parsers.append(FreeProxyParser('http://free-proxy-list.net', timeout=timeout))
+        parsers.append(ProxyForEuParser('http://proxyfor.eu/geo.php', 1.0, timeout=timeout))
+        parsers.append(RebroWeeblyParser('http://rebro.weebly.com', timeout=timeout))
         # parsers.append(SamairProxyParser('http://samair.ru/proxy/time-01.htm'))
```

Comment:
SemairProxy seems to be working now  - we should uncomment this line.
Also if timeout is working it won't cause issues in the future